io_buffer.c

```diff
@@ -2528,6 +2531,31 @@ rb_io_buffer_pread(VALUE self, VALUE io, rb_off_t from, size_t length, size_t of
     return rb_thread_io_blocking_region(io_buffer_pread_internal, &argument, descriptor);
 }
 
+/*
+ *  call-seq: pread(io, from, length, [offset]) -> numbytes
```

Comment:
I don't like abbreviations and try to avoid them. Therefore, I want to find a better result than `numbytes`. I don't have a strong opinion but maybe `number of bytes read` or `number of bytes written`?